website/content/en/docs/overview/cheat-sheet.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Cheat Sheet"
+linkTitle: "Cheat Sheet"
+weight: 3
+description: >
+    Operator-SDK Cheat Sheet commands and operations
+---
+
+Following a cheat sheet with the options and helpers for projects which are built with SDK and are respecting its proposed layout.
+
+## Base commands and options 
+
+| Command   | Description  |
+|-------|-----------|
+| `operator-sdk init`          | To initialize a operator project in the current directory |
+| `operator-sdk init --plugins=<plugin-key>`          | To initialize a operator project in the current directory using an specific plugin. For you are able to check the available plugins you can run `oparator-sdk --help`. E.g (`operator-sdk init --plugins=helm`)|
+| `operator-sdk create api [flags]`          | Let you create your own APIs with its [GKV][gkvs] by [Extendig the Kubernetes API with CustomResourceDefinitions][extend-k8s-api], or let you use external or core-types as well as generate their respectively [controllers][controllers-k8s-doc]|
+| `operator-sdk create webhook [flags]`          | To scaffold [Webhooks][webhooks-k8s-doc] for the APIs declared onn the project. Note that only Go-based project supports this option currently.|
+| `make docker-build IMG=<some-registry>/<project-name>:<tag>`          | Build the operator image      |
+| `make docker-build docker-push IMG=<some-registry>/<project-name>:<tag>`      | Build and push the operator image for your registry  |
+| `make install`         | Install the CRDs into the cluster |
+| `make uninstall`         | Uninstall the CRDs into the cluster |
+| `make run`         | Run your controller locally and outside of the cluster. Note that, this will run in the foreground, so switch to a new terminal if you want to leave it running. |
```

Comment:
```suggestion
| `make run`         | Run your controller locally and outside of the cluster. Note that this will run in the foreground, so switch to a new terminal if you want to leave it running. |
```